erpnext/selling/report/test_style/test_style.py

```
@@ -0,0 +1,474 @@
+
+	statuses = ["Completed", "Pending", "In Progress", "Delivered", "Cancelled"]
+
+	data = []
+	start_date = datetime(2025, 1, 1)
+
+	# N = 1_00_000  # Number of rows to generate
+	N = 50  # Number of rows to generate
+
+	for i in range(1, N):
+		# Generate random transaction date
+		days_offset = random.randint(0, 364)
```

Ruff (0.14.13)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)